### PR TITLE
O3-916 Get rid of Refapp 2.x links in top-left menu

### PR DESCRIPTION
**Pull request by 99sumedha**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Screenshots
image

## Related Issue

https://issues.openmrs.org/browse/O3-916

## Other

*None.*

### CODE REVIEW

File: packages/esm-home-app/src/index.ts [hunk 1]
```diff
@@ -41,47 +41,47 @@ function setupOpenMRS() {
     extensions: [
       {
         id: 'active-visits-link',
-        slot: 'app-menu-slot',
+        slot: '',
```
Comment (brandones) [inline packages/esm-home-app/src/index.ts:44]: You can just delete these lines completely. I will improve the documentation to cover this.

Comment (99sumedha) [reply packages/esm-home-app/src/index.ts:44]: Hi @brandones, sure i will update the changes.

**Comment by 99sumedha** [comment]
Hi @brandones, PR updated :)

**Review by brandones — APPROVED** [review]

**Comment by brandones** [comment]
Thanks @99sumedha !

**Pull request merged**